### ISSUE TITLE
Is it possible to store the needed session data in localStroage instead of a cookie? 

### ISSUE THREAD

**Issue by mooglin** [socketstream/socketstream#423]
- [ ] options for setting localStorage / add-on
- [ ] PhoneGap example

I'm trying to use socketstream with phonegap. The session becomes lost when the app is restarted, the cookie does not persist. I was thinking I could store the needed data in localStroage because that does seem to persist. 

Is this possible? What data needs to be stored/updated? 

The goal is autologin / persistent login between app restarts, not just page reloads, which already works.

**Comment by paulbjensen** [comment]
I believe that @kulicuu is looking into this issue, is that right?

**Comment by kulicuu** [comment]
Yes, although only generally at this point. The engine.io migration turned out not to require such a major change.

**Comment by thepian** [comment]
@mooglin I will be fixing this in the coming weeks for `0.5`, any experience you would like to share on this?
